test/operators.js

```diff
@@ -473,23 +473,40 @@ test('live votesFor', (t) => {
         db.query(
           and(votesFor(postMsg.key)),
           live({ old: true }),
```

Comment:
@arj03 Check below. This was actually using the API wrong, but it happened to function correctly!

Before, it was 

```js
pull( 
  db.query(stuff, toPullStream(), pull.drain(etc)
)
```

When it should have been:


```js
pull( 
  db.query(stuff, toPullStream()), 
  pull.drain(etc)
)
```

Reply:
Yeah I noticed :-)